Solutions/DNS Essentials/NgrokReverseProxyOnNetwork

```diff
@@ -0,0 +1,30 @@
+id: 50b0dfb7-2c94-4eaf-a332-a5936d78c263
+name: Ngrok Reverse Proxy on Network (ASIM DNS Solution)
+description: |
+  'This detection identifies the top 4 Ngrok domains from DNS resolution. Ngrok’s reverse proxy can
+tags:
+  - Schema: ASimDns
+    SchemaVersion: 0.1.6
+requiredDataConnectors: []
+tactics:
+  - Command And Control
+relevantTechniques:
+  - T1572
+  - T1090
+  - T1102
+query: |
+  // List of Ngrok-like domains
+  let NgrokDomains = dynamic(["ngrok.com", "ngrok.io", "ngrok", "tunnel.com", "korgn", "lennut.com"]);
+  _Im_Dns
+  | where TimeGenerated > ago(1h) // Logs from last hour
+  | where isnotempty(DnsQuery) // Non-empty DNS queries
+  | where DnsQuery has_any (NgrokDomains) // Queries containing Ngrok domains
```

Comment:
I would do summarize count() by DnsQuery. just to limit duplicate rows with same entry

Reply:
Each event has an separate id in dns events